aspnetcore/tutorials/grpc/grpc-start.md

```diff
@@ -76,19 +76,19 @@ From a terminal, run the following commands:
 
 The preceding commands use the [.NET Core CLI](/dotnet/core/tools/dotnet) to create a gRPC service.
 
-## Open the project
+### Open the project
 
 From Visual Studio, select **File > Open**, and then select the *GrpcGreeter.sln* file.
 
 <!-- End of VS tabs -->
 
 ---
 
-## Run and test the service
+### Test the service
 
 # [Visual Studio](#tab/visual-studio)
 
-* Ensure the **GrpcGreeter.Server** is set as the Startup Project and press Ctrl+F5 to run the gRPC service without the debugger.
+* Ensure the **GrpcGreeter.Server** is set as the Startup Project and press Ctrl+F5 to run the gRPC service without the debugger. Visual Studio launches a browser. Close the browser window.
```

Comment:
Does the sample open a browser window? It shouldn't.

Reply:
Oops, forgot to remove that. That's from VS generating new launch settings, etc.